packages/gamut-labs/src/experimental/GlobalFooter/FooterNavLinks/CompanyLinks.tsx

```diff
@@ -0,0 +1,192 @@
+import { Anchor, Box } from '@codecademy/gamut';
+import { theme } from '@codecademy/gamut-styles';
+import styled from '@emotion/styled';
+import React from 'react';
+
+import { FooterHeading } from '../FooterHeading';
+import { LinkArea } from '../FooterLinkArea';
+import { FooterLinkItem } from '../FooterLinkItem';
+import { SocialMediaLinks } from './SocialMediaLinks';
+
+export type CompanyLinksProps = {
+  /**
+   * Geographic region of the user viewing the footer, such as "IN" or "US".
+   */
+  userGeo: string;
+};
+
+const CompanyLinksContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+
+  ${theme.breakpoints.sm} {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+`;
```

Comment:
Maybe GridBox?

Also we may want to set these to `minmax(0, 1fr)` for overflow safety.

```suggestion
<GridBox gridTemplateColumns={['repeat(2, minmax(0, 1fr))', , repeat(3, minmax(0, 1fr))']} />
```